packages/golden-layout/.gitignore

```diff
@@ -1,6 +1,3 @@
-node_modules
 .idea
 npm-debug.log
-.DS_Store
-dist
 css
```

Comment:
Can remove this if we just ignore `css` from the root. Then the whole file can be deleted